error: use of `extend` instead of `append` for adding the full range of a second vector
  --> tests/ui/extend_with_drain.rs:8:5
   |
LL |     vec2.extend(vec1.drain(..));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D clippy::extend-with-drain` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::extend_with_drain)]`
help: try
   |
LL |     vec2.append(&mut vec1);
   |     ~~~~~~~~~~~~~~~~~~~~~~

error: use of `extend` instead of `append` for adding the full range of a second vector
  --> tests/ui/extend_with_drain.rs:13:5
   |
LL |     vec4.extend(vec3.drain(..));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
help: try
   |
LL |     vec4.append(&mut vec3);
   |     ~~~~~~~~~~~~~~~~~~~~~~

error: use of `extend` instead of `append` for adding the full range of a second vector
  --> tests/ui/extend_with_drain.rs:17:5
   |
LL |     vec11.extend(return_vector().drain(..));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
help: try
   |
LL |     vec11.append(&mut return_vector());
   |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error: use of `extend` instead of `append` for adding the full range of a second vector
  --> tests/ui/extend_with_drain.rs:48:5
   |
LL |     y.extend(ref_x.drain(..));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^
   |
help: try
   |
LL |     y.append(ref_x);
   |     ~~~~~~~~~~~~~~~

error: aborting due to 4 previous errors

